ext/github.py

```diff
@@ -10,19 +10,22 @@
 from cog import Cog
 
 
-class GithubIntegration(Cog):
+headers = {
+            "Content-Type":"application/json",
+            "Accept": "application/json"
+        }
+
+
+class Github(Cog):
     """
     A class containing commands/functions for Github Integration.
     """
     def __init__(self, bot):
         # Save our bot for later use
+        self.bot.http_session    = aiohttp.ClientSession()
```

Comment:
This is not necessary, at all. See [this](https://github.com/justalemon/Chomusuke/blob/master/bot.py#L31) for more information.